### PR TITLE
feat: add notes/ delete notes

### PR DESCRIPTION
**Pull request by techieshubham30**

### CODE REVIEW

File: src/pages/LandingPage/Signup/Signup.js [hunk 1]
```diff
@@ -37,57 +37,67 @@ const Signup = () => {
...
-        </div>
+    <div className="wrapper">
+      <div className="signup-container">
+        <h1 className="signup-title">SIGN UP</h1>
+        <form className="display-flex-column">
+          <div className="display-flex-row">
+            <label className="form-label">Email</label>
+            <input
+              type="email"
+              class="form-field"
```
Comment (Sen-442b) [inline src/pages/LandingPage/Signup/Signup.js:48]: You can use react className attribute here

**Review by Sen-442b — APPROVED** [review]
Folder structure and  code looks clean

File: src/pages/LandingPage/Signup/Signup.js (hunk 1 above)
Comment (techieshubham30) [reply src/pages/LandingPage/Signup/Signup.js:48]: yes, I just forgot to update it, thanks for pointing it out.

**Review by bipin7yadav — APPROVED** [review]
ui looks good . Good to go

**Pull request merged**